concent_api/common/tests/test_api_view.py

```diff
@@ -218,10 +219,23 @@ def setUp(self):
         message_timestamp = get_current_utc_timestamp() + deadline_offset
         compute_task_def = ComputeTaskDefFactory()
         compute_task_def['deadline'] = message_timestamp
+        want_to_compute_task = WantToComputeTaskFactory(
+            provider_public_key=encode_hex(PROVIDER_PUBLIC_KEY),
+        )
+        want_to_compute_task = load(
```

Comment:
Why not using `.sign_message()` to achieve this?

Reply:
I thought that it's a method from our `ConcentIntegrationTestCase`. I'm gone use our `sign_message()` from `common.helpers` instead of GM once, since we use it everywhere.

Reply:
Done